### DISCUSSION TITLE
no access to copilot

### DISCUSSION THREAD

**Discussion by koolcandy** [community/community#113944] [GitHub Education]
### Hi everyone,

I have no access to github copilot , I had submit the ticket three days ago ,but I get no answer
the link is https://support.github.com/ticket/personal/0/2689775
can anyone help me ? I really need to use copilot , its really important for my daily coding

**Comment by AlgoOy** [comment]
![image](https://github.com/community/community/assets/44948890/ad445084-077b-48f3-9a56-68db970bcb38)
I have the same problem.

**Comment by devashish2024** [comment]
Hey there, I'll be trying to help you.

You will get 1 year of GitHub copilot access with Github Student Pack. Once approved, you might have to wait a few days for the perks to process.

After it's approved, like you got it yet, you need to go here https://github.com/github-copilot/signup.
You might see a screen **You are invited to use GitHub Copilot for free**, meaning you have access to github copilot. You need to just enable it on the link and that's all, then go to your settings, copilot to see if its active now.

> if I resolved your issue, please consider this as answer, ty!

**Reply by koolcandy** [reply]
thank you for answer !
however , I already have the Github Student Pack , I had use it for 2 months,  but 3 days ago I found that I have no access to it , it just return 403, then I submit the ticket, but got no answer

**Reply by AlgoOy** [reply]
hi @ashishagarwal2023 When I open this link, it is redirected to https://github.com/settings/copilot
![image](https://github.com/community/community/assets/44948890/408c8115-2511-4fe9-99ee-f82a911eaec8)

**Reply by devashish2024** [reply]
I'm not sure but your account appears to be under a university, so there might be some different things on it. For most peoples including me (I used this link when having same issue 5 days ago when I got approved) it worked.

**Reply by devashish2024** [reply]
@koolcandy Can you give me a image of your registeration? You can get it from [here](https://education.github.com/discount_requests/application).
![image](https://github.com/community/community/assets/83082760/7ffc195e-f0e7-4fa7-8a51-a1c6771fc690)

After being approved for GitHub Student Dev Pack, if you didn't still got a purple **Approved**, it means that your perks are being proccesed.

> It will take some days, for me I waited 4 to get them but sometimes they gets long on holidays.

After you've been accepted, try going to the same link [here](https://github.com/github-copilot/signup) and see if it works.

If this solved your problem, consider marking it as answer.

![image](https://github.com/community/community/assets/83082760/a1047635-0150-4d30-aecb-76009e43c419)

(Not my photo but that is what you see when you have got access for free)

**Reply by koolcandy** [reply]
![截屏2024-03-24 23 19 51](https://github.com/community/community/assets/133551107/29655578-6cbe-429b-b9d6-73f548586b69)
![截屏2024-03-24 23 20 29](https://github.com/community/community/assets/133551107/608c1fa5-b55f-49f0-bcf1-8d932a0027c4)

**Reply by koolcandy** [reply]
![截屏2024-03-24 23 22 00](https://github.com/community/community/assets/133551107/f6a20520-e058-40f6-b280-d3583e0a26f5)

**Reply by devashish2024** [reply]
Ugh, did you tried GitHub copilot extension on VSCode or any IDE and after authenticating with GitHub, did it work or said to buy?

**Reply by koolcandy** [reply]
It ask me to connect support, then i submit the ticket, got no answer :(

**Reply by doveppp** [reply]
I have the same problem!

**Reply by devashish2024** [reply]
Try to give it a few days of time. It should work!

**Reply by koolcandy** [reply]
thanks to github staffs，my account get right!

**Reply by doveppp** [reply]
help by github support?

**Reply by koolcandy** [reply]
yes ，I submit the ticket then staff help me to slove the problem

**Reply by doveppp** [reply]
May I ask where this ticket is from? I have already contacted GitHub support 4 days ago and have not received a response yet.

edit:
Ah, I understand now. It refers to the support from GitHub. I'll wait a few more days then.

**Reply by koolcandy** [reply]
> May I ask where this ticket is from? I have already contacted GitHub support 4 days ago and have not received a response yet.
> 
> edit:
> Ah, I understand now. It refers to the support from GitHub. I'll wait a few more days then.

you should submit the ticket ,go to support.github.com

**Reply by doveppp** [reply]
Of course, I submitted the ticket four days ago, it looks like there should be a response tomorrow.

**Reply by koolcandy** [reply]
> Of course, I submitted the ticket four days ago, it looks like there should be a response tomorrow.

the same , maybe today!

**Comment by devashish2024** [comment]
> ### Hi everyone,
> I have no access to github copilot , I had submit the ticket three days ago ,but I get no answer the link is https://support.github.com/ticket/personal/0/2689775 can anyone help me ? I really need to use copilot , its really important for my daily coding

Your ticket is not publicly accessible, sorry.
